packages/core/src/browser/shell/application-shell.ts

```diff
@@ -96,13 +96,25 @@ export class DockPanelRenderer implements DockLayout.IRenderer {
         @inject(TabBarToolbarRegistry) protected readonly tabBarToolbarRegistry: TabBarToolbarRegistry,
         @inject(TabBarToolbarFactory) protected readonly tabBarToolbarFactory: TabBarToolbarFactory,
         @inject(BreadcrumbsRendererFactory) protected readonly breadcrumbsRendererFactory: BreadcrumbsRendererFactory,
+        @inject(CorePreferences) protected readonly corePreferences: CorePreferences
```

Comment:
We should document this as a breaking change:

- https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#property-injection